src/components/section/blog-section.json

```diff
@@ -16,6 +16,9 @@
       "type": "component",
       "repeatable": false,
       "component": "elements.section-intro"
+    },
+    "pagination": {
+      "type": "boolean"
```

Comment:
Change to select with : infinite, pagination and off

Reply:
> Change to select with : infinite, pagination and off

1. created pagination collection have three state such like infinite, pagination and off
2. added pagination into blog && project

Reply:
Hummm, why so complicated ? I've asking for a type enum with [infinite, pagination and off].
I will check it